src/main/java/org/sagebionetworks/web/client/widget/entity/DoiWidget.java

```diff
@@ -104,7 +101,16 @@ public void onFailure(Throwable caught) {
 				}
 			}
 		});
+	}	
+	
+	public void configureDoi() {
```

Comment:
is this method referenced?  it looks more like a clear() method rather than a configureDoi().